src/features/market-status/components/MarketStatus.tsx

```diff
@@ -13,24 +13,26 @@ type MarketStatusProps = {
   keyStats: IKeyStats | null;
 };
 
-export const MarketStatus: FC<MarketStatusProps> = ({
-  stock,
-  keyStats
-}: any) => {
-  const UKTime = formatDate(stock.latestUpdate);
-  const USTime = formatDate(UKTime).toLocaleString("en-US", {
-    timeZone: "America/New_York"
-  });
-  const correctFormat = moment(new Date(USTime)).format("lll");
-  const marketStatus = keyStats.open === null ? "Market Closed" : "Market Open";
+export const MarketStatus: FC<MarketStatusProps> = ({ stock, keyStats }) => {
+  const UKTime = stock && formatDate(stock.latestUpdate);
```

Comment:
Not this PR:
The creation of a `UKTime` to make a `USTime` to make a `correctFormat` is very long-winded.
Can't you just do:
```ts
const formattedDate = stock && moment(stock.latestUpdate).format("lll");
```
? Or am I missing something.

Reply:
Very good point - will change in a new PR